gramps_webapi/api/resources/util.py

```diff
@@ -453,13 +453,16 @@ def get_place_profile_for_handle(
 
 
 def get_person_profile_for_object(
-    db_handle: DbReadBase, person: Person, args: list, locale: GrampsLocale = glocale
+    db_handle: DbReadBase,
+    person: Person,
+    args: list,
+    locale: GrampsLocale = glocale,
+    name_format: str = None,
```

Comment:
This type hint is wrong since `None` is not a string. `str | None`, and please make sure that `from __future__ import annotations` is imported at the top of the file.